types/node-forge/index.d.ts

```diff
@@ -43,7 +43,9 @@ declare module "node-forge" {
             privateKey: Key;
         }
 
+		function pemToDer(pem: PEM): util.ByteStringBuffer;
         function privateKeyToPem(key: Key, maxline?: number): PEM;
+		function privateKeyInfoToPem(key: Key, maxline?: number): PEM;
```

Comment:
Indent using spaces instead of tabs.

Reply:
Sorry, I wasn't paying attention after submitting this. I haven't submitted very many PRs before, so it looks like I need to update and move the the state of this PR back to reviewer